src/App.js

```diff
@@ -3,90 +3,101 @@ import download from "downloadjs";
+   */
+  neo4jService._getNeo4jDriver();
+
+  const [layers, setLayers] = React.useState([]);
+  const [collapsed, setCollapse] = React.useState(false);
 
-	const [layers, setLayers] = React.useState([]);
+  const addLayer = (layer) => {
+    setLayers([...layers, layer]);
+  };
 
-	const addLayer = (layer) => {
-		setLayers([...layers, layer]);
-	}
+  const updateLayer = (layer) => {
+    const updatedLayers = layers.map((currentLayer) => {
+      if (currentLayer.ukey === layer.ukey) {
+        return layer;
+      }
+      return currentLayer;
+    });
 
-	const updateLayer = (layer) => {
-		const updatedLayers = layers.map(currentLayer => {
-			if (currentLayer.ukey === layer.ukey) {
-				return layer;
-			}
-			return currentLayer;
-		});
+    setLayers(updatedLayers);
+  };
 
-		setLayers(updatedLayers);
-	}
+  const removeLayer = (key) => {
+    const filteredLayers = layers.filter((layer) => layer.ukey !== key);
 
-	const removeLayer = (key) => {
-		const filteredLayers = layers.filter(layer => layer.ukey !== key);
+    setLayers(filteredLayers);
+  };
 
-		setLayers(filteredLayers);
-	}
+  const saveConfigToFile = (e) => {
+    const config = JSON.stringify(layers);
+    const fileName = "neomap_config.json";
+    download(config, fileName, "application/json");
+    e.preventDefault();
+  };
 
-	const saveConfigToFile = (e) => {
-		const config = JSON.stringify(layers);
-		const fileName = "neomap_config.json";
-		download(config, fileName, "application/json");
-		e.preventDefault();
-	};
+  const loadConfigFromFile = (e) => {
+    const fileSelector = document.createElement("input");
+    fileSelector.setAttribute("type", "file");
+    fileSelector.click();
+    fileSelector.onchange = (ev) => {
+      const file = ev.target.files[0];
+      const fileReader = new FileReader();
+      fileReader.onloadend = (e) => {
+        const content = e.target.result;
+        try {
+          const loadedLayers = JSON.parse(content);
+          setLayers(loadedLayers);
+        } catch (err) {
+          // TODO: Build error UI
+          console.log("Failed to load and parse data from file", err);
+        }
+      };
+      fileReader.readAsText(file);
+    };
+    e.preventDefault();
+  };
 
-	const loadConfigFromFile = (e) => {
-		const fileSelector = document.createElement('input');
-		fileSelector.setAttribute('type', 'file');
-		fileSelector.click();
-		fileSelector.onchange = (ev) => {
-			const file = ev.target.files[0];
-			const fileReader = new FileReader();
-			fileReader.onloadend = (e) => {
-				const content = e.target.result;
-				try {
-					const loadedLayers = JSON.parse(content);
-					setLayers(loadedLayers);
-				} catch (err) {
-					// TODO: Build error UI
-					console.log('Failed to load and parse data from file', err);
-				}
-			};
-			fileReader.readAsText(file);
-		};
-		e.preventDefault();
-	};
+  const defaultMapOffset = 30;
```

Comment:
Are you sure these numbers work on different screen size?

Reply:
not much but 
![image](https://user-images.githubusercontent.com/33401112/135757320-b9eed8a0-96cb-4785-a56d-526c78f1d397.png)

Reply:
i'll try to come up with something better

Reply:
Super small window:
![image](https://user-images.githubusercontent.com/33401112/135763561-7ed22110-442c-4620-9ee2-178c3e070151.png)

Fullscreen window at 1920*1080 display
![image](https://user-images.githubusercontent.com/33401112/135763521-1c5a173e-36c8-4852-9886-a4b00fec2203.png)

cant test 2k/4k resolutions

Reply:
without fixing width to 60px sidebar'd been hiding a bit too much
![image](https://user-images.githubusercontent.com/33401112/135763919-777f4766-49d2-4fa6-8bad-d1565d98a931.png)